client_integration_test.go

```diff
@@ -1489,6 +1489,49 @@ func TestClientReadFrom(t *testing.T) {
 	}
 }
 
+func TestClientReadFromSectionReader(t *testing.T) {
+	const (
+		packetSize = 1024
+		bufsize    = 4 * packetSize
+		skip       = 100
+		filesize   = bufsize - skip
+	)
+
+	sftp, cmd := testClient(t, READWRITE, NODELAY,
```

Comment:
This function call is more readable when all on one line, and splitting the options out is awkward.
https://github.com/golang/go/wiki/CodeReviewComments#line-length

Reply:
Done.